public/public.php

```diff
@@ -467,30 +467,33 @@ public function list_post_attachments_with_sources( $post_id = 0 ) {
 
 		if ( ! empty( $attachments ) ) {
 			ISC_Log::log( sprintf( 'going through %d attachments', count( $attachments ) ) );
-			$atts = array();
+			$atts = [];
 			foreach ( $attachments as $attachment_id => $attachment_array ) {
 				$use_standard_source = Standard_Source::use_standard_source( $attachment_id );
 				$source              = self::get_image_source_text( $attachment_id );
 
 				// check if source of own images can be displayed
-				if ( ( $use_standard_source == '' && $source == '' ) || ( $use_standard_source != '' && $exclude_standard ) ) {
-					if ( $use_standard_source != '' && $exclude_standard ) {
+				if ( ( $use_standard_source === '' && $source === '' ) || ( $use_standard_source !== '' && $exclude_standard ) ) {
+					if ( $use_standard_source !== '' && $exclude_standard ) {
 						ISC_Log::log( sprintf( 'image %d: "own" sources are excluded', $attachment_id ) );
 					} else {
 						ISC_Log::log( sprintf( 'image %d: skipped because of empty source', $attachment_id ) );
 					}
 					unset( $atts[ $attachment_id ] );
```

Comment:
**Suggestion:** Consider using `isset()` instead of `!== ''` for checking if a variable is set or not. This is a more reliable way to check if a variable has a value or not. [best practice]
```suggestion
    if ( ( !isset($use_standard_source) && !isset($source) ) || ( isset($use_standard_source) && $exclude_standard ) ) {
      if ( isset($use_standard_source) && $exclude_standard ) {
        ISC_Log::log( sprintf( 'image %d: "own" sources are excluded', $attachment_id ) );
      } else {
        ISC_Log::log( sprintf( 'image %d: skipped because of empty source', $attachment_id ) );
      }
      unset( $atts[ $attachment_id ] );
    }
```